hmy/api_backend.go

```diff
@@ -127,53 +130,63 @@ func (b *APIBackend) ProtocolVersion() int {
 // Filter related APIs
 
 // GetLogs ...
+// TODO: this is not implemented or verified yet for harmony.
 func (b *APIBackend) GetLogs(ctx context.Context, blockHash common.Hash) ([][]*types.Log, error) {
 	// TODO(ricl): implement
 	return nil, nil
 }
 
 // HeaderByHash ...
+// TODO: this is not implemented or verified yet for harmony.
 func (b *APIBackend) HeaderByHash(ctx context.Context, blockHash common.Hash) (*types.Header, error) {
 	// TODO(ricl): implement
 	return nil, nil
 }
 
 // ServiceFilter ...
+// TODO: this is not implemented or verified yet for harmony.
```

Comment:
For the APIs that I've already added TODO, no need to add the TODO twice.

Reply:
done